cg/services/sequencing_qc_service/sequencing_qc_service.py

```diff
@@ -6,10 +11,25 @@
     get_sequencing_quality_check_for_case,
     get_sample_sequencing_quality_check,
 )
+from cg.store.store import Store
+
+LOG = logging.getLogger(__name__)
 
 
 class SequencingQCService:
 
+    def __init__(self, store: Store):
+        self.store = store
+
+    def run_sequencing_qc(self) -> None:
+        """Run sequencing QC for all pending or failed cases."""
+        cases: list[Case] = self.store.get_cases_for_sequencing_qc()
+        for case in cases:
+            passes_qc: bool = self.case_pass_sequencing_qc(case)
+            qc_status: SequencingQCStatus = qc_bool_to_status(passes_qc)
+            self.store.update_sequencing_qc_status(case=case, status=qc_status)
```

Comment:
We wanted to put the sequencingqc status on the sample, not the case if I remember correctly.

Reply:
This is the case level QC, so it belongs on the case. It is separate from the sample QC

Reply:
Cases are not sequenced, samples are. We can check whether all samples in the case pass the sequencing qc. 
If this is to be on the case level, which we did not say in the refinement, then it should be named differently. 

Perhaps: ready_to_analyse

Reply:
My understanding from the refinement was -> put sequencing qc on the sample.
Then a case can be analysed when all the samples pass the qc. but maybe I misunderstood.

Reply:
The app-tag on the sample determines the amount of reads it has to have.

Priority is on both the sample AND the case level, beats me what either of those mean now... will have to check the relationship there... 

Yes, I would not call it "sequencing" on a case level.

I would use terms that are sequencing related on sample and analysis related on case

Then let sequencing related attributes on the sample determine the case attribute, since the samples will be re-used and the number of reads should not make a difference since that is determined by the apptag (sample level).

Reply:
We renamed it to `aggregated_sequencing_qc` instead of `sequencing_qc`. The `ready_for_analysis` is misleading since other criteria factor in such as whether the data needs to be retrieved from pdc. What do you think?

Reply:
We majority support for this naming, closing this to move forward getting rid of the technical debt. We can rename it later